src/app/pages/ArticlePage/Byline/index.test.tsx

```diff
@@ -1,35 +1,39 @@
 /* eslint-disable no-template-curly-in-string */
 import React, { PropsWithChildren } from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, act } from '@testing-library/react';
 import Byline from '.';
 import { ServiceContextProvider } from '../../../contexts/ServiceContext';
 import ArticleTimestamp from '../../../legacy/containers/ArticleTimestamp';
 import {
   bylineWithNoRole,
   bylineWithNoAuthor,
   bylineWithNameAndRole,
+  bylineWithLink,
 } from './fixture';
+import ThemeProvider from '../../../components/ThemeProvider';
 
 interface Props {
+  service: any;
   fixture: any;
-  service: string;
 }
 
 const FixtureByline = ({
-  fixture,
   service,
+  fixture,
   children,
-}: PropsWithChildren<Props>) => {
-  return (
-    <ServiceContextProvider service={service}>
+}: PropsWithChildren<Props>) => (
+  <ServiceContextProvider service={service}>
+    <ThemeProvider service={service}>
       <Byline blocks={fixture}>{children}</Byline>
-    </ServiceContextProvider>
-  );
-};
+    </ThemeProvider>
+  </ServiceContextProvider>
+);
 
 describe('Byline', () => {
-  it('Should render Byline correctly when only required data is passed', () => {
-    render(<FixtureByline fixture={bylineWithNameAndRole} service="news" />);
+  it('Should render Byline correctly when only required data is passed', async () => {
+    await act(async () => {
+      render(<FixtureByline fixture={bylineWithNameAndRole} service="news" />);
+    });
```

Comment:
I think all of these `await act(...)` wrappers can be removed and you can use the new `render` and `screen` functions from `react-testing-library-with-providers`. I used it in another PR for reference: https://github.com/bbc/simorgh/blob/d27736e2a824a93640ac126055bf8f8acce9f115/src/app/components/EmbedConsentBanner/ConsentBanner.test.tsx

Reply:
Done, also made use of JoRo's fancy custom react testing library with our context providers https://github.com/bbc/simorgh/tree/latest/src/app/components#custom-react-testing-library